README.md

```diff
@@ -60,7 +63,35 @@ tox
 
 ## Runtime Environment
 
-Install (from sources) system-wide:
+### Install pre-requirements
+```shell
+sudo yum -y install epel-release
```

Comment:
The dev installations do not use sudo but yum directly. Consistency would be nice.

Reply:
Ok, will drop the `sudo`.